scribble-lib/scribble/private/tag.rkt

```diff
@@ -0,0 +1,39 @@
+#lang scheme/base
+
+;; It might make sense to make these functions public, but since they weren't originally,
+;; I am going to keep them in the private folder for now.
+
+(require racket/list
+         scribble/core
+         "../tag.rkt")
+
+(provide (all-defined-out))
```

Comment:
Nope. Predicates and contracts that are defined _in_ the module can be used before they are defined. But ones defined in _other_ modules cannot be.